lib/shopify_app/managers/scripttags_manager.rb

```diff
@@ -24,6 +24,7 @@ def self.build_src(scripttags, domain)
     attr_reader :required_scripttags, :shop_domain
 
     def initialize(scripttags, shop_domain)
+      ShopifyApp::Logger.deprecated("The ScripttagsManager will become deprecated in an upcoming version", "22.0.0")
```

Comment:
Is on initialize enough? Since this is a class we could use [this](https://api.rubyonrails.org/classes/ActiveSupport/Deprecation/MethodWrapper.html#method-i-deprecate_methods) to send a message every time one of its methods gets called?

Reply:
I really like that idea to use `ActiveSupport::Deprecation::MethodWrapper`, however I'd vote we do that after we merge what we have here.